draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/NavigationBar.tsx

```diff
@@ -22,7 +23,7 @@ type Props = {
   loading?: boolean
   colorScheme?: ColorScheme
   badgeHref?: string
-  onNavigationChange: NavigationChange
+  onNavigationChange?: NavigationChange
```

Comment:
This was a weird one. There are a bunch of usages of this component that don't have this prop, but they weren't resulting in TS errors until we added the `withDeprecatedComponent` wrapping.